pydm/widgets/timeplot.py

```diff
@@ -147,30 +252,49 @@ def initialize_for_designer(self):
         # This function gets called by PyDMTimePlot's designer plugin.
         self.redraw_timer.setSingleShot(True)
 
+    def getPlotByTimestamps(self):
+        """
+        Whether the graph will show the moving timestamps as the x-axis (default), or the relative time after the
+        starting time.
+
+        :return: True if the x-axis will show the moving timestamps (default); or the relative time as the x-axis
```

Comment:
We use Numpy style for the docstrings... please adjust.

Reply:
Fixed.